metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/profiler.py

```diff
@@ -231,7 +231,7 @@ def generate_wu_from_profile_requests(
             # We don't add to the profiler state if we only do table level profiling as it always happens
             if self.state_handler and not request.profile_table_level_only:
                 self.state_handler.add_to_state(
-                    dataset_urn, int(datetime.utcnow().timestamp() * 1000)
+                    dataset_urn, int(datetime.now().timestamp() * 1000)
```

Comment:
Why is this change needed ? I think, we should always persist UTC timestamps. In fact, if last_altered from database query is non-utc, we should convert it to UTC on database side OR locally in python code before comparing with timestamp from profiling state. WDYT ?

Reply:
in general, we should use timezone-aware datetimes

```py
datetime.now(timezone.utc)
```

however, because we're calling .timestamp() immediately, these two are basically the same - probably ok for now

```
>>> datetime.now().timestamp()
1673267826.735305
>>> datetime.now(timezone.utc).timestamp()
1673267828.30003
>>>
```

Reply:
Ohh, got it.